results/fp8-blackwell-5090/20260719-154503/triton-benchmark-smoke-final/triton_neuron_kernel_benchmark.md

```
@@ -0,0 +1,48 @@
+| 32 | 4096 | if | stable_bfloat16 | training_forward_backward | 2.280 | 1.911-2.649 | 57.493 | 2.517/260.000 |
+| 32 | 4096 | if | mp_plan_float8_e4m3fn_fp16 | training_forward_backward | 1.980 | 1.829-2.131 | 66.198 | 3.282/260.000 |
+| 32 | 4096 | if | mp_plan_float8_e4m3fn_bf16 | training_forward_backward | 1.820 | 1.795-1.845 | 72.014 | 3.532/260.000 |
+| 32 | 4096 | if | mp_plan_float8_e4m3fn_fp32 | training_forward_backward | 2.045 | 2.029-2.061 | 64.090 | 3.532/260.000 |
+| 32 | 4096 | if | mp_plan_float8_e5m2_fp16 | training_forward_backward | 1.960 | 1.799-2.121 | 66.869 | 3.532/262.000 |
+| 32 | 4096 | if | mp_plan_float8_e5m2_bf16 | training_forward_backward | 1.824 | 1.795-1.853 | 71.861 | 3.532/260.000 |
+| 32 | 4096 | if | mp_plan_float8_e5m2_fp32 | training_forward_backward | 2.033 | 2.015-2.051 | 64.481 | 3.532/260.000 |
+| 32 | 4096 | if | stable_fp32 | training_forward_backward | 1.608 | 1.459-1.757 | 81.527 | 5.033/262.000 |
+
+## Failures
+
+- <code>T=32 N=4096 neuron=if variant=mp_plan_float8_e4m3fn_fp8 process=inference_forward: RuntimeError: Triton FP8 IF forward is unavailable for torch.float8_e4m3fn with compute_dtype=&#x27;fp8&#x27;: triton.compiler.errors.CompilationError: at 11:8: def _fp8_neuron_forward_probe_kernel(     x_ptr,     y_ptr,     N: tl.constexpr,     compute_dtype: tl.constexpr,     BLOCK: tl.constexpr, ):     offsets = tl.arange(0, BLOCK)     mask = offsets &lt; N     x = tl.load(x_ptr + offsets, mask=mask, other=0.0).to(compute_dtype)     v = tl.full([BLOCK], 0.125, dtype=compute_dtype)         ^ &#x27;triton._C.libtriton.ir.builder&#x27; object has no attribute &#x27;get_fp8e4nv&#x27;</code>
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)